### PR TITLE
Add DTS issue reporting

### PR DESCRIPTION
**Pull request by DaniilKl**

### CODE REVIEW

**Comment by artur-rs** [comment]
@DaniilKl looks good, thank you. For now, merge is blocked by the PR: https://github.com/Dasharo/dasharo-issues/pull/744

File: docs/dasharo-tools-suite/overview.md [hunk 1]
```diff
@@ -24,7 +24,35 @@ Thank you for using Dasharo Tools Suite. If you have encountered any problems
...
+
+It is not always possible to copy/paste logs from DTS. In such cases, photos or
+screenshots are also acceptable. Just imagine you are a professional
+photographer trying to capture another natural miracle in the best quality
+possible.
+
+There is a list of common tools which can be used to get more information about
+your case:
+
+* `dmidecode`: will provide detailed information about your system;
```
Comment (macpijan) [inline docs/dasharo-tools-suite/overview.md:45]: I am not sure in which cases these commands would be useful when reporting bugs agains DTS. Have we looked at some past bugs to see which information was missing? If that would be required, it should go into template, but I am not so convinced.

File: docs/dasharo-tools-suite/overview.md [hunk 2]
```diff
@@ -24,7 +24,35 @@ Thank you for using Dasharo Tools Suite. If you have encountered any problems
 with this system or would like to provide feedback for us - please open an issue
 on [Dasharo
 issues](https://github.com/Dasharo/dasharo-issues/issues?q=is%3Aopen+is%3Aissue+label%3ADasharoToolsSuite).
+Dasharo Tools Suite has its own label: `DasharoToolsSuite` and issue template
+for bugs: `DTS bug report`. Other types of issues in the repository are generic
```
Comment (macpijan) [inline docs/dasharo-tools-suite/overview.md:28]: Simply provide a direct link to the report: https://github.com/Dasharo/dasharo-issues/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml and instruct to choose `Dasharo Tools Suite` as a component

File: docs/dasharo-tools-suite/overview.md [hunk 3]
```diff
@@ -24,7 +24,35 @@ Thank you for using Dasharo Tools Suite. If you have encountered any problems
...
+and can be used with the `DasharoToolsSuite` label.
 
 And if you have already used this system and would be interested in supporting
 the project, please check the
 [How to support us?](../osf-trivia-list/dts.md#how-to-support-us) section.
+
+## Gathering additional information and providing logs
```
Comment (macpijan) [inline docs/dasharo-tools-suite/overview.md:35]: I would omit this section. If really required, I would move it to bug report.

File: docs/dasharo-tools-suite/overview.md (hunk 1 above)
Comment (DaniilKl) [reply docs/dasharo-tools-suite/overview.md]: It is useful if some extra info is needed while reporting an issue, but I guess it is issue specific and should not be added to this documentation. Amended here b07c88d5d094dcab8b4de9bbec664d15942e09fc.

File: docs/dasharo-tools-suite/overview.md (hunk 2 above)
Comment (DaniilKl) [reply docs/dasharo-tools-suite/overview.md]: Link added here b07c88d5d094dcab8b4de9bbec664d15942e09fc.

File: docs/dasharo-tools-suite/overview.md (hunk 3 above)
Comment (DaniilKl) [reply docs/dasharo-tools-suite/overview.md]: Amended here b07c88d5d094dcab8b4de9bbec664d15942e09fc.

**Review by macpijan — APPROVED** [review]

**Pull request merged**